src/components/AxiosInterceptor/helpers/useSetNativeAuthInterceptors.ts

```diff
@@ -49,7 +49,7 @@ export const useSetNativeAuthInterceptors = () => {
 
         let bearerToken = newToken;
         const hasAllowedURLs =
-          config.baseURL?.includes('extras-api.multiversx.com') ||
+          config.baseURL?.includes('extras-lite-api.multiversx.com') ||
```

Comment:
nu va mai exista URL-ul vechi?

Reply:
We will migrate to a public extras API, yes